app/src/main/java/com/h/pixeldroid/objects/Instance.kt

```diff
@@ -0,0 +1,12 @@
+package com.h.pixeldroid.objects
+
+data class Instance (
+//    val description: String,
```

Comment:
No commented code in master please

Reply:
Pk t'as commenté ces champs d'ailleurs? Si tu les utilises pas, tu peux les ignorer, mais une Instance contient ces trucs